### PR TITLE
Feature/sg 591 basic toolkit

### PR DESCRIPTION
**Pull request by shaydeci**

### CODE REVIEW

File: documentation/source/QuickstartBasicToolkit.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
+# The Basic Skills of SG
+
+Learn the basics of model development with SuperGradients. Researchers and machine learning engineers should start here.
```
Comment (ofrimasad) [inline documentation/source/QuickstartBasicToolkit.md:3]: please change:
In this tutorial we will go over all of the besic functionalities of SuperGradients very briefly. 
Go over the next sections to quickly learn how to train, test and predict using SuperGradients. 
Check-out our extended tutorials on the verious features you can find in SuperGradients, and task specific guids.

File: documentation/source/QuickstartBasicToolkit.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+
+
+<details>
+    <summary>1. Train a Model</summary>
+
+##1. Train a Model
```
Comment (ofrimasad) [inline documentation/source/QuickstartBasicToolkit.md:9]: I dont think we need the sub-title if we have the folding sub-title. 
But even if you wish to keep it, please add a space after the ## , so it will be recognized as a title (in all titles)

File: documentation/source/QuickstartBasicToolkit.md [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+from super_gradients.training import Trainer, models
+from super_gradients.training.metrics.classification_metrics import Accuracy, Top5
+from super_gradients.training.dataloaders.dataloaders import cifar10_train, cifar10_val
+from super_gradients.training.utils.distributed_training_utils import setup_device
+```
+
+1. Call [setup_device()](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/device.md) according to your available hardware and needs:
+
+```python
+setup_device("cpu")
```
Comment (ofrimasad) [inline documentation/source/QuickstartBasicToolkit.md:24]: what other values are available here (please add in a comment in the code).
you don't want the user to have to go into the link and back

File: documentation/source/QuickstartBasicToolkit.md [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+```
+
+2. Instantiate a [Trainer]() object #TODO: ADD TRAINER API LINK
+```python
+
+trainer = Trainer(experiment_name="my_cifar_experiment", ckpt_root_dir="/path/to/checkpoints_directory/")
```
Comment (ofrimasad) [inline documentation/source/QuickstartBasicToolkit.md:30]: don't we have to call `deci_trainer.init_trainer()` first?

File: documentation/source/QuickstartBasicToolkit.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+```
+
+3. [Instantiate a model](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/models.md):
+```python
+model = models.get(Models.RESNET18, num_classes=10)
+```
+
+4. Define [metrics](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/Metrics.md) and other [training parameters](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/recipes/training_hyperparams/default_train_params.yaml):
+```
```
Comment (ofrimasad) [inline documentation/source/QuickstartBasicToolkit.md:39]: missing 'python' for formatting

File: documentation/source/QuickstartBasicToolkit.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+1. Call [setup_device()](https://github.com/Deci-AI/super-gradients/blob/master/documentation/source/device.md) according to your available hardware and needs:
+
+```python
+setup_device("cpu")
+```
+
+2. Instantiate a [Trainer]() object #TODO: ADD TRAINER API LINK
+```python
+trainer = Trainer(experiment_name="test_my_cifar_experiment", ckpt_root_dir="/path/to/checkpoints_directory/")
```
Comment (ofrimasad) [inline documentation/source/QuickstartBasicToolkit.md:86]: same here: `deci_trainer.init_trainer()`

**Review by ofrimasad — COMMENTED** [review]
please see my comment in line.
I understand we want to be brief in this tutorial, but lets make it useful

File: documentation/source/QuickstartBasicToolkit.md (hunk 1 above)
Comment (shaydeci) [reply documentation/source/QuickstartBasicToolkit.md]: Fixed https://github.com/Deci-AI/super-gradients/pull/718/commits/d9199dfa4e657910a027f3acfd768c85b6a8fa28

File: documentation/source/QuickstartBasicToolkit.md (hunk 2 above)
Comment (shaydeci) [reply documentation/source/QuickstartBasicToolkit.md]: Fixed https://github.com/Deci-AI/super-gradients/commit/d9199dfa4e657910a027f3acfd768c85b6a8fa28

File: documentation/source/QuickstartBasicToolkit.md (hunk 3 above)
Comment (shaydeci) [reply documentation/source/QuickstartBasicToolkit.md:31]: Added in latest commit.

File: documentation/source/QuickstartBasicToolkit.md (hunk 4 above)
Comment (shaydeci) [reply documentation/source/QuickstartBasicToolkit.md:49]: Fixed https://github.com/Deci-AI/super-gradients/commit/d9199dfa4e657910a027f3acfd768c85b6a8fa28.

File: documentation/source/QuickstartBasicToolkit.md (hunk 5 above)
Comment (shaydeci) [reply documentation/source/QuickstartBasicToolkit.md]: Fixed.

File: documentation/source/QuickstartBasicToolkit.md (hunk 6 above)
Comment (shaydeci) [reply documentation/source/QuickstartBasicToolkit.md:122]: Fixed everywhere.

**Review by ofrimasad — APPROVED** [review]
LGTM
